src/__tests__/util.test.js

```diff
@@ -47,7 +47,7 @@ describe('Util function tests', () => {
       expect(constructTypeQuery(type).replace(/\s/g, '')).toBe(query);
     });
 
-    // TODO: The rest of the assembleQueries tests.
+    // Next: The rest of the assembleQueries tests.
```

Comment:
NooO!!!!

I wanted to use comment tags that have semantic highlighting 😭 

(I don't personally mind the warnings. They're a pretty shade of yellow, don't you think?)

Reply:
lovely shade fine with me!!!